lago/prefix.py

```diff
@@ -104,6 +104,14 @@ def __init__(self, prefix):
         self.paths = paths.Paths(self._prefix)
         self._virt_env = None
         self._metadata = None
+        self._internal_repo = None
+
+    def _get_internal_repo (self):
+        return self._internal_repo
+
+    def _set_internal_repo (self, path):
+        self._internal_repo = path
+
```

Comment:
this setter and getter are not needed, you can just use instance._internal_repo to access the property